lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
 
+const _ = require('lodash');
 const path = require('path');
 const BbPromise = require('bluebird');
 
 module.exports = {
   create() {
     this.serverless.cli.log('Creating Stack...');
     const stackName = `${this.serverless.service.service}-${this.options.stage}`;
+    let stackTags = { STAGE: this.options.stage };
```

Comment:
Any user defined config should only be set in the update-stack action so the create can succeed even if the update stack. Can you remove the changes from createStack.js so the tags are properly set in the update